std/net/curl.d

```diff
@@ -2807,6 +2808,35 @@ struct HTTP
         }
     }
 
+    /// Clear ContentLength depending on method.
```

Comment:
This is internal function as far as I can tell. Then do *not* use DDoc comment just `//` is enough.